priv/posts/machine-learning/20210519123456_machine-learning-wg-updates-may.md

```diff
@@ -0,0 +1,86 @@
+{
+  "title": "Machine Learning WG updates - May 2021",
+  "authors": ["José Valim", "Sean Moriarity", "Jonatan Kłosko", "Tom Rutten"],
```

Comment:
I am thinking it would probably be best if the author is "The Machine Learning WG". Thoughts?

Reply:
Sounds good to me!